--- apachegrep~	2005-10-11 23:05:16.000000000 +0200
+++ apachegrep	2007-03-26 14:57:35.000000000 +0200
@@ -68,7 +68,7 @@
    }
    else {
       my $fileopener = ($arg =~ /\.bz2$/) ? [ '-|', "bzcat $arg" ]
-                     : ($arg =~ /\.gz$/ ) ? [ '-|', "gzcat $arg" ]
+                     : ($arg =~ /\.gz$/ ) ? [ '-|', "zcat $arg" ]
                      :                      [ '<',  $arg                ]
                      ;
 
